src/ServiceBusEvent.php

```diff
@@ -240,11 +239,7 @@ protected function getPayload(): array
      */
     private function generateUUID(string $key): string
     {
-        $uuid = Uuid::uuid5(Uuid::NAMESPACE_DNS, 'php.net');
-
-        Log::info('Generating UUID', ['tag' => 'ServiceBus', 'id' => $uuid->toString(), 'key' => $key]);
-
-        return $uuid->toString();
+        return Uuid::uuid5(Uuid::NAMESPACE_DNS, 'php.net')->toString();
```

Comment:
I think we are using v4 UUIDs in the bus. please correct me if I'm wrong @zebroc @dele454

Reply:
v4 UUID seems like a good idea since this pkg is used across projects.

Reply:
If we use the $key for uuid5 generation, it _must_ be a domain/host name.

Reply:
Seems to be UUID4, if this is anything to by.
https://github.com/RingierIMU/rsb-go-lib/blob/master/base/types.go

Reply:
lol, docs above this even say: UUID4